src/Microsoft.Dnx.Tooling/Restore/RestoreCommand.cs

```diff
@@ -414,7 +426,6 @@ public RestoreCommand(FrameworkName fallbackFramework)
                             {
                                 VersionRange = new SemanticVersionRange(project.Version)
                             };
-                            Reports.Information.WriteLine(string.Format("Graph for {0} on {1}", runtimeContext.FrameworkName, runtimeContext.RuntimeName));
```

Comment:
Because this trace was a little too noisy, or at least it would get very noisy as more and more runtime.json packages exist. @lodejard even said that from the start it was a debugging trace.